Tests/RxBlockingTests/Observable+BlockingTest.swift

```diff
@@ -17,11 +17,11 @@ class ObservableBlockingTest : RxTest {
 
 extension ObservableBlockingTest {
     func testToArray_empty() {
-        XCTAssert(try! Observable<Int>.empty().toBlocking().toArray() == [])
+        XCTAssertEqual(try! Observable<Int>.empty().toBlocking().toArray(), [])
```

Comment:
Why the styling change here? 
Moving from XCTAssert with == to XCTAssertEqual ?